json_merger/conflict.py

```diff
@@ -93,5 +94,27 @@ def with_prefix(self, root_path):
         return Conflict(self.conflict_type, root_path + self.path, self.body)
 
     def to_json(self):
-        """Deserializes conflict to a JSON object."""
-        return json.dumps([self.conflict_type, self.path, self.body])
+        """Deserializes conflict to a JSON object.
+        
+        It returns `json-patch <https://tools.ietf.org/html/rfc6902>`_ format.
+            - REORDER, SET_FIELD become "op": "replace"
+            - MANUAL_MERGE, ADD_BACK_TO_HEAD become "op": "add"
+            - Path array becomes `json-pointer 
+            <https://tools.ietf.org/html/rfc6901>`_
+            - Original conflict type is added to "$type"
+        """
+        # map ConflictType to json-patch operator
+        if self.conflict_type in ('REORDER', 'SET_FIELD'):
+            op = 'replace'
+        elif self.conflict_type in ('MANUAL_MERGE', 'ADD_BACK_TO_HEAD'):
+            op = 'add'
+        else:
+            op = 'remove'
+        # stringify path array
+        path = '/' + '/'.join(str(el) for el in self.path)
```

Comment:
I am not sure if path has a `number` or a `string` for `index`, I am gonna check the examples or try to reproduce it but maybe @kaplun knows?

So if it doesn't we can remove this conversion.